packages/api/apollo-server/queries/user.ts

```diff
@@ -0,0 +1,35 @@
+import { users, match } from '../mock-data'
+
+export interface IGetUser {
+  id?: String
+  utorID?: String
+  email?: String
+}
+
+// get one user based on id, utorID or email
+export const getUser = async (_: any, args: IGetUser) => {
+  const resultUser = users.find(user => user.id === args.id)
+  return resultUser
+}
+
+export interface IGetUsers {
+  users: {
```

Comment:
I don't think it's necessary for this to be an array. Just have one criteria and keep it simple.